2_process/src/munge_GFv1_catchments.R

```diff
@@ -11,7 +11,9 @@ find_intersecting_hru <- function(prms_line,prms_hrus){
   
   # Transform PRMS segment and HRU's into a consistent projection
   # using Albers Equal Area Conic CONUS
-  prms_line_proj <- sf::st_transform(prms_line, 5070)
+  prms_line_proj <- sf::st_transform(prms_line, 5070) %>%
+    sf::st_buffer(5) %>%
```

Comment:
Could you add the reasoning behind needing the buffer here to the comment above?

Reply:
Yep, good idea. I added a comment to explain how the buffer is used in https://github.com/USGS-R/drb-network-prep/pull/7/commits/910cc43d4be1e70ce66e6ab894de9637d4d78289. Thanks for the nudge!